---
layout: default
title: Luo Lab Publications
---
<p><span style="font-weight:bold">Selected Publications</span></p>
<p>Luo, C.*, Liu, H.*, Xie, F.*, et al. <a href="https://www.biorxiv.org/content/10.1101/2019.12.11.873398v1" target="_blank">Single nucleus multi-omics links human cortical cell regulatory genome diversity to disease risk variants. </a><strong><em>bioRxiv </strong></em> (2019) </p>
<p>Lee, D.*, Luo, C.*, Zhou, J.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/31501549" target="_blank">Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. </a><strong><em>Nature Methods </strong></em> (2019) </p>
<p>Luo, C. et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30237449%5BPMID%5D" target="_blank">Robust single-cell DNA methylome profiling with snmC-seq2.</a><strong><em> Nature Communications.</em></strong> (2018) </p>
<p>Luo, C.*, Keown, C.L.*, et al.  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28798132%5BPMID%5D" target="_blank">Single-cell methylomes identify neuronal subtypes and regulatory elements in mammalian cortex.</a><strong><em> Science.</em></strong> (2017) </p>
<p>Mo, A.*, Mukamel, E.A.*, Davis, F.P.*, Luo, C.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26087164" target="_blank">Epigenomic Signatures of Neuronal Diversity in the Mammalian Brain. </a><strong><em>Neuron.</strong></em> (2015) </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">BRAIN Initiative Cell Census Network Publications (2020) </span></p>
<p>Liu, H.*, Zhou, J.*, et al. <a href="https://www.biorxiv.org/content/10.1101/2020.04.30.069377v1" target="_blank"> DNA Methylation Atlas of the Mouse Brain at Single-Cell Resolution. </a><strong><em>bioRxiv </strong></em> (2020) </p>
<p>Bakken, T.E., et al. <a href="https://www.biorxiv.org/content/10.1101/2020.03.31.016972v2" target="_blank"> Evolution of cellular diversity in primary motor cortex of human, marmoset monkey, and mouse. </a><strong><em>bioRxiv </strong></em> (2020) </p>
<p>Yao, Z.*, Liu, H.*, Xie, F.*, Fischer, S.*, et al. <a href="https://www.biorxiv.org/content/10.1101/2020.02.29.970558v2" target="_blank"> An integrated transcriptomic and epigenomic atlas of mouse primary motor cortex cell types. </a><strong><em>bioRxiv </strong></em> (2020) </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">Neural Epigenomics</span></p>
<p>Li, J.*, Pinto-Duarte, A.*, et al. <a href="https://www.biorxiv.org/content/10.1101/2019.12.20.883694v1" target="_blank">Polycomb-mediated repression compensates for loss of postnatal DNA methylation in excitatory neurons. </a><strong><em>bioRxiv. </strong></em> (2019) </p>
<p>Lavery, L.A., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/32159514" target="_blank">Losing Dnmt3a dependent methylation in inhibitory neurons impairs neural function by a mechanism impacting Rett syndrome. </a><strong><em>Elife. </strong></em> (2020) </p>
<p>Luo, C.*, Lee, Q.Y.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30644360%5BPMID%5D" target="_blank">Global DNA methylation remodeling during direct reprogramming of fibroblasts to neurons.</a><strong><em> Elife.</em></strong> (2019) </p>
<p>Luo, C.*, Hajkova, P.*, Ecker, J.R. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30262495%5BPMID%5D" target="_blank">Dynamic DNA methylation: In the right place at the right time.</a><strong><em> Science.</em></strong> (2018) </p>
<p>Sabbagh, M.F., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30188322%5BPMID%5D" target="_blank">Transcriptional and epigenomic landscapes of CNS and non-CNS vascular endothelial cells.</a><strong><em> Elife.</em></strong> (2018) </p>
<p>Luo, C.*, Lancaster, M.A.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28009303%5BPMID%5D" target="_blank">Cerebral Organoids Recapitulate Epigenomic Signatures of the Human Fetal Brain.</a><strong><em> Cell Reports.</em></strong> (2016) </p>
<p>Mo, A., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26949250%5BPMID%5D" target="_blank">Epigenomic landscapes of retinal rods and cones.</a><strong><em> Elife.</em></strong> (2016) </p>
<p>Luo, C., Ecker, J.R. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26045424" target="_blank"> Exceptional epigenetics in the brain. </a><strong><em>Science.</strong></em> (2015) </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">Plant Epigenomics</span></p>
<p>Widiez, T., Symeonidi, A., <span style="font-weight:bold">Luo, C.</span>, Lam, E., Lawton, M. and Rensing, S.A. <a href="https://www.ncbi.nlm.nih.gov/pubmed/24779858/" target="_blank">The chromatin landscape of the moss Physcomitrella patens and its dynamics during development and drought stress.</a> (2014) <strong><em>Plant J.</strong></em> 79: 67-81. </p>
<p><span style="font-weight:bold">Luo, C.*</span>, Dong, J.*, Zhang, Y. and Lam, E.  <a href="https://www.ncbi.nlm.nih.gov/pubmed/25191327" target="_blank">Decoding the role of chromatin architecture in development: coming closer to the end of the tunnel.</a> (2014) <strong><em>Front. Plant Sci.</strong></em> 5:374. *Co-first author. </p>
<p><span style="font-weight:bold">Luo, C.</span> and Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/24478015" target="_blank">Quantitatively profiling genome-wide patterns of histone modifications in Arabidopsis thaliana using ChIP-seq.</a> (2014) <strong><em>Methods Mol Biol.</strong></em> 1112: 177-193. </p>
<p><span style="font-weight:bold">Luo, C.</span>, Sidote, D.J., Zhang, Y., Kerstetter, R.A., Michael, T.P. and Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22962860" target="_blank">Integrative analysis of chromatin states in Arabidopsis identified potential regulatory mechanisms for Natural Antisense Transcript production.</a> (2012) <strong><em>Plant J.</strong></em> 73: 77-90. </p>
<p>Vaquero-Sedas, M.I., <span style="font-weight:bold">Luo, C.</span> and Vega-Palas, M.A. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22855559" target="_blank">Analysis of the epigenetic status of telomeres by using ChIP-seq data.</a> (2012) <strong><em>Nucleic Acids Res.</strong></em> 40: e163. </p>
<p>Amini, A., <span style="font-weight:bold">Luo, C.</span> and Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/21181538" target="_blank">Chromatin Beacons: Global sampling of chromatin physical properties using chromatin charting lines.</a> (2011) <strong><em>Methods Mol Biol.</strong></em> 701: 301-14. </p>
<p><span style="font-weight:bold">Luo, C.</span>, and Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/20444227" target="_blank">ANCORP: a high resolution approach that generated distinct chromatin state model from multiple genome-wide datasets.</a> (2010) <strong><em>Plant J.</strong></em> 63: 339-351. </p>
<p>Lam, E., <span style="font-weight:bold">Luo, C.</span> and Watanabe, N. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19327981" target="_blank">Charting functional and physical properties of chromatin in living cells.</a> (2009) <strong><em>Curr Opin Genet Dev.</strong></em> 19, 135-141. </p>
<p><span style="font-weight:bold">Luo, C.*</span>, Durgin, B.G.*, Watanabe, N.*, and Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19825647" target="_blank">Defining the Functional Network of Epigenetic Regulators in Arabidopsis thaliana.</a> (2009) <strong><em>Mol. Plant.</strong></em> 2, 661-674. *Co-first author. </p>
<p><span style="font-weight:bold">Luo, C.</span> and Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19588104" target="_blank">Chromatin Charting: Global mapping of epigenetic effects.</a> (2009) <strong><em>Methods Mol Biol.</strong></em> 553, 127-139. </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">Other Publications</span></p>
<p>Chin, C.S.*, Peluso, P.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27749838%5BPMID%5D" target="_blank">Phased diploid genome assembly with single-molecule real-time sequencing.</a><strong><em> Nature Methods.</em></strong> (2016) </p>
<p>Wu, J., Okamura, D., Li, M., Suzuki, K., <span style="font-weight:bold">Luo, C.</span>, Ma, L., He, Y., Li, Z., Benner, C., Tamura, I., Krause, M.N., Nery, J.R., Du, T., Zhang, Z., Hishida, T., Takahashi, Y., Aizawa, E., Kim, N.Y., Lajara, J., Guillen, P., Campistol, J.M., Esteban, C.R., Ross, P.J., Saghatelian, A., Ren, B., Ecker, J.R., Izpisua Belmonte, J.C. (2015). <a href="https://www.ncbi.nlm.nih.gov/pubmed/25945737/" target="_blank">An alternative pluripotent state confers interspecies chimaeric competency.</a> <strong><em> Nature.</strong></em>  521: 316-321. </p>
<p>Rotter, D., Bharti, A.K., Li, H.M., <span style="font-weight:bold">Luo, C.</span>, Bonos, S.A., Bughrara, S., Jung, G., Messing, J., Meyer, W.A., Rudd, S., Warnke, S.E., and Belanger, F.C. <a href="https://www.ncbi.nlm.nih.gov/pubmed/17497174" target="_blank">Analysis of EST sequences suggests recent origin of allotetraploid colonial and creeping bentgrasses.</a> (2007) <strong><em>Mol Genet Genomics.</strong></em> 278, 197-209. </p>
