---
layout: default
title: Luo Lab Publications
---
<img width=200 class="projectmedia" src="cellgenomics_mar2022.gif">
<img width=200 class="projectmedia" src="nature_oct2021.png">
<img width=200 class="projectmedia" src="science_sep2018.png">
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">Selected Publications</span></p>
<p>Luo, C.*, Liu, H.*, Xie, F.*, et al. <a href="https://www.cell.com/cell-genomics/fulltext/S2666-979X(22)00027-1" target="_blank">Single nucleus multi-omics identifies human cortical cell regulatory genome diversity. </a><strong><em>Cell Genomics </strong></em> (2022) </p>
<p>Lee, D.*, Luo, C.*, Zhou, J.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/31501549" target="_blank">Simultaneous profiling of 3D genome structure and DNA methylation in single human cells. </a><strong><em>Nature Methods </strong></em> (2019) </p>
<p>Luo, C. et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30237449%5BPMID%5D" target="_blank">Robust single-cell DNA methylome profiling with snmC-seq2.</a><strong><em> Nature Communications.</em></strong> (2018) </p>
<p>Luo, C.*, Keown, C.L.*, et al.  <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28798132%5BPMID%5D" target="_blank">Single-cell methylomes identify neuronal subtypes and regulatory elements in mammalian cortex.</a><strong><em> Science.</em></strong> (2017) </p>
<p>Mo, A.*, Mukamel, E.A.*, Davis, F.P.*, Luo, C.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26087164" target="_blank">Epigenomic Signatures of Neuronal Diversity in the Mammalian Brain. </a><strong><em>Neuron.</strong></em> (2015) </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">COVID-19</span></p>
<p>Bloom, J.S., et al., <a href="https://pubmed.ncbi.nlm.nih.gov/34211145/" target="_blank"> Massively scaled-up testing for SARS-CoV-2 RNA via next-generation sequencing of pooled and barcoded nasal and saliva samples. </a><strong><em>Nat Biomed Eng </strong></em> (2021) </p>
<p>Guo, L., et al., <a href="https://www.biorxiv.org/content/10.1101/2020.08.15.252510v1" target="_blank"> Rapid cost-effective viral genome sequencing by V-seq. </a><strong><em>bioRxiv </strong></em> (2020) </p>
<p>Guo, L.*, Boocock, J.*, et al., <a href="https://www.medrxiv.org/content/10.1101/2020.09.15.20194712v1" target="_blank"> Genomic epidemiology of the Los Angeles COVID-19 outbreak. </a><strong><em>medRxiv </strong></em> (2020) </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">BRAIN Initiative Cell Census Network</span></p>
<p>BRAIN Initiative Cell Census Network (BICCN). <a href="https://pubmed.ncbi.nlm.nih.gov/34616075/" target="_blank"> A multimodal cell census and atlas of the mammalian primary motor cortex. </a><strong><em>Nature </strong></em> (2021) </p>
<p>Liu, H.*, Zhou, J.*, et al. <a href="https://pubmed.ncbi.nlm.nih.gov/34616061/" target="_blank"> DNA Methylation Atlas of the Mouse Brain at Single-Cell Resolution. </a><strong><em>Nature </strong></em> (2021) </p>
<p>Bakken, T.E., et al. <a href="https://pubmed.ncbi.nlm.nih.gov/34616062/" target="_blank"> Evolution of cellular diversity in primary motor cortex of human, marmoset monkey, and mouse. </a><strong><em>Nature </strong></em> (2021) </p>
<p>Yao, Z.*, Liu, H.*, Xie, F.*, Fischer, S.*, et al. <a href="https://pubmed.ncbi.nlm.nih.gov/34616066/" target="_blank"> A transcriptomic and epigenomic cell atlas of the mouse primary motor cortex. </a><strong><em>Nature </strong></em> (2021) </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">Mammalian Epigenomics</span></p>
<p>Gao. C., et al. <a href="https://pubmed.ncbi.nlm.nih.gov/33875866/" target="_blank">Iterative single-cell multi-omic integration using online learning. </a><strong><em>Nat Biotechnol. </strong></em> (2021) </p>
<p>Evrony, G.D.*, Hinch, A.G.*, Luo, C.*, et al. <a href="https://pubmed.ncbi.nlm.nih.gov/33722077/" target="_blank">Applications of Single-Cell DNA Sequencing. </a><strong><em>Annu Rev Genomics Hum Genet. </strong></em> (2021) </p>
<p>Armand, E.J.*, Li, J.*, Xie, F.*, et al. <a href="https://pubmed.ncbi.nlm.nih.gov/33412093/" target="_blank">Single-Cell Sequencing of Brain Cell Transcriptomes and Epigenomes. </a><strong><em>Neuron. </strong></em> (2021) </p>
<p>He, Y., et al. <a href="https://pubmed.ncbi.nlm.nih.gov/32728242/" target="_blank">Spatiotemporal DNA methylome dynamics of the developing mouse fetus. </a><strong><em>Nature. </strong></em> (2020) </p>
<p>Li, J.*, Pinto-Duarte, A.*, et al. <a href="https://www.biorxiv.org/content/10.1101/2019.12.20.883694v1" target="_blank">Polycomb-mediated repression compensates for loss of postnatal DNA methylation in excitatory neurons. </a><strong><em>bioRxiv. </strong></em> (2019) </p>
<p>Lavery, L.A., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/32159514" target="_blank">Losing Dnmt3a dependent methylation in inhibitory neurons impairs neural function by a mechanism impacting Rett syndrome. </a><strong><em>Elife. </strong></em> (2020) </p>
<p>Luo, C.*, Lee, Q.Y.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30644360%5BPMID%5D" target="_blank">Global DNA methylation remodeling during direct reprogramming of fibroblasts to neurons.</a><strong><em> Elife.</em></strong> (2019) </p>
<p>Luo, C.*, Hajkova, P.*, Ecker, J.R. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30262495%5BPMID%5D" target="_blank">Dynamic DNA methylation: In the right place at the right time.</a><strong><em> Science.</em></strong> (2018) </p>
<p>Sabbagh, M.F., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=30188322%5BPMID%5D" target="_blank">Transcriptional and epigenomic landscapes of CNS and non-CNS vascular endothelial cells.</a><strong><em> Elife.</em></strong> (2018) </p>
<p>Luo, C.*, Lancaster, M.A.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=28009303%5BPMID%5D" target="_blank">Cerebral Organoids Recapitulate Epigenomic Signatures of the Human Fetal Brain.</a><strong><em> Cell Reports.</em></strong> (2016) </p>
<p>Mo, A., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=26949250%5BPMID%5D" target="_blank">Epigenomic landscapes of retinal rods and cones.</a><strong><em> Elife.</em></strong> (2016) </p>
<p>Wu, J.*, Okamura, D.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/25945737/" target="_blank">An alternative pluripotent state confers interspecies chimaeric competency.</a> <strong><em> Nature.</strong></em> (2015) </p>
<p>Luo, C., Ecker, J.R. <a href="https://www.ncbi.nlm.nih.gov/pubmed/26045424" target="_blank"> Exceptional epigenetics in the brain. </a><strong><em>Science.</strong></em> (2015) </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">Plant Epigenomics</span></p>
<p>Widiez, T.*, Symeonidi, A.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/24779858/" target="_blank">The chromatin landscape of the moss Physcomitrella patens and its dynamics during development and drought stress.</a><strong><em>Plant J.</strong></em> (2014) </p>
<p>Luo, C.*, Dong, J.*, et al., <a href="https://www.ncbi.nlm.nih.gov/pubmed/25191327" target="_blank">Decoding the role of chromatin architecture in development: coming closer to the end of the tunnel.</a><strong><em>Front. Plant Sci.</strong></em> (2014) </p>
<p>Luo, C., Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/24478015" target="_blank">Quantitatively profiling genome-wide patterns of histone modifications in Arabidopsis thaliana using ChIP-seq.</a><strong><em>Methods Mol Biol.</strong></em> (2014) </p>
<p>Luo, C., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22962860" target="_blank">Integrative analysis of chromatin states in Arabidopsis identified potential regulatory mechanisms for Natural Antisense Transcript production.</a><strong><em>Plant J.</strong></em> (2012) </p>
<p>Vaquero-Sedas, M.I., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/22855559" target="_blank">Analysis of the epigenetic status of telomeres by using ChIP-seq data.</a><strong><em>Nucleic Acids Res.</strong></em> (2012) </p>
<p>Amini, A., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/21181538" target="_blank">Chromatin Beacons: Global sampling of chromatin physical properties using chromatin charting lines.</a><strong><em>Methods Mol Biol.</strong></em> (2011) </p>
<p>Luo, C., Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/20444227" target="_blank">ANCORP: a high resolution approach that generated distinct chromatin state model from multiple genome-wide datasets.</a><strong><em>Plant J.</strong></em> (2010) </p>
<p>Lam, E., et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19327981" target="_blank">Charting functional and physical properties of chromatin in living cells.</a><strong><em>Curr Opin Genet Dev.</strong></em> (2009) </p>
<p>Luo, C.*, Durgin, B.G.*, Watanabe, N.*, and Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19825647" target="_blank">Defining the Functional Network of Epigenetic Regulators in Arabidopsis thaliana.</a><strong><em>Mol. Plant.</strong></em> (2009) </p>
<p>Luo, C., Lam, E. <a href="https://www.ncbi.nlm.nih.gov/pubmed/19588104" target="_blank">Chromatin Charting: Global mapping of epigenetic effects.</a><strong><em>Methods Mol Biol.</strong></em> (2009) </p>
<div class="bigspacer"> </div>

<p><span style="font-weight:bold">Other Publications</span></p>
<p>Chin, C.S.*, Peluso, P.*, et al. <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=27749838%5BPMID%5D" target="_blank">Phased diploid genome assembly with single-molecule real-time sequencing.</a><strong><em> Nature Methods.</em></strong> (2016) </p>
<p>Rotter, D., Bharti, A.K., Li, H.M., <span style="font-weight:bold">Luo, C.</span>, Bonos, S.A., Bughrara, S., Jung, G., Messing, J., Meyer, W.A., Rudd, S., Warnke, S.E., and Belanger, F.C. <a href="https://www.ncbi.nlm.nih.gov/pubmed/17497174" target="_blank">Analysis of EST sequences suggests recent origin of allotetraploid colonial and creeping bentgrasses.</a> (2007) <strong><em>Mol Genet Genomics.</strong></em> 278, 197-209. </p>
